src/components/DatePicker/RelativeDateRange.jsx

```diff
@@ -220,13 +233,22 @@ export const RelativeDateRange = ({
     onChange,
     onSubmit,
     timezoneName,
+    isFutureDatesAllowed = false, // New prop
 }) => {
     const handleChange = (e) => {
         const rangeName = e.target.value;
         const range = handlers[rangeName](timezoneName);
         onChange(rangeName, range);
     };
 
+    const filterFutureDates = (options) => {
+        if (!isFutureDatesAllowed) {
+            // Filter out future date options
+            return options.filter((option) => !option.value.includes("next"));
```

Comment:
can you check with Future date instead of checking just "next" text
const futureDates = [options.NEXT_QUARTER, ....] and check if not exist here